cscs-checks/apps/quantumespresso/quantumespresso_check.py

```diff
@@ -65,26 +91,37 @@ def __init__(self, scale, variant):
             'maint': {
                 'small': {
                     'dom:mc': {'time': (115.0, None, 0.05, 's')},
-                    'daint:mc': {'time': (115.0, None, 0.10, 's')}
+                    'daint:mc': {'time': (115.0, None, 0.10, 's')},
+                    'eiger:mc': {'time': (66.0, None, 0.10, 's')}
                 },
                 'large': {
-                    'daint:mc': {'time': (115.0, None, 0.10, 's')}
+                    'daint:mc': {'time': (115.0, None, 0.10, 's')},
+                    'eiger:mc': {'time': (53.0, None, 0.10, 's')}
                 }
             },
             'prod': {
                 'small': {
                     'dom:mc': {'time': (115.0, None, 0.05, 's')},
-                    'daint:mc': {'time': (115.0, None, 0.10, 's')}
+                    'daint:mc': {'time': (115.0, None, 0.10, 's')},
+                    'eiger:mc': {'time': (66.0, None, 0.10, 's')}
                 },
                 'large': {
-                    'daint:mc': {'time': (115.0, None, 0.10, 's')}
+                    'daint:mc': {'time': (115.0, None, 0.10, 's')},
+                    'eiger:mc': {'time': (53.0, None, 0.10, 's')}
                 }
             }
         }
 
         self.reference = references[variant][scale]
         self.tags |= {'maintenance' if variant == 'maint' else 'production'}
 
+    @rfm.run_before('run')
+    def set_task_distribution(self):
+        self.job.options = ['--distribution=block:block']
+
+    @rfm.run_before('run')
+    def set_cpu_binding(self):
+        self.job.launcher.options = ['--cpu-bind=cores']
```

Comment:
Are these meant for eiger only or are they ok for the rest of the systems, too?

Reply:
I tested them on Eiger only for the moment, since they were boosting a bit the performance.

Reply:
I am asking because as you have coded it now, it will run with these options on every system. The CI does not check for performance in this case, because the test is marked as non-strict (which we could change tbh).

Reply:
Thanks for checking @vkarak: as far as I could test on Dom and Piz Daint, the additional Slurm options do not harm the performance of this check. We have been using the same options already with the CP2K check on Dom and Piz Daint.